### PR TITLE
add ibmz hypershift e2e job

### PR DESCRIPTION
**Pull request by LiangquanLi930**

### CODE REVIEW

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-ipi-ovn-hypershift-mce-ibmz-guest-critical-f7

**Comment by LiangquanLi930** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-ipi-ovn-hypershift-mce-ibmz-guest-critical-f7

**Comment by LiangquanLi930** [comment]
@liangxia PTAL Thanks.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml [hunk 1]
```diff
@@ -617,6 +617,61 @@ tests:
...
     - chain: openshift-e2e-test-hypershift-qe-mgmt
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-mce
+- as: aws-ipi-ovn-hypershift-mce-ibmz-guest-critical-f7
+  cron: 3 10 6,13,20,27 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@amd64 and @aws-ipi and @network-ovnkubernetes and @critical
+        and @hypershift-hosted and not @fips'
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml:627]: The other tags in `E2E_RUN_TAGS` can be skipped (as they can dynamically detected) except `@critical`.

Comment (LiangquanLi930) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml]: Thanks, done

**Pull request merged**